packages/sirius-web/backend/sirius-web-graphql/src/main/java/org/eclipse/sirius/web/graphql/datafetchers/representation/RepresentationMetadataDescriptionDataFetcher.java

```diff
@@ -64,23 +92,27 @@ public CompletableFuture<IRepresentationDescription> get(DataFetchingEnvironment
         CompletableFuture<IRepresentationDescription> result = Mono.<IRepresentationDescription>empty().toFuture();
 
         RepresentationMetadata representationMetadata = environment.getSource();
-        if (Objects.equals(PropertiesEventProcessorFactory.DETAILS_VIEW_ID, representationMetadata.getDescriptionId())) {
+        if (Objects.equals(PropertiesEventProcessorFactory.DETAILS_VIEW_ID, representationMetadata.getDescriptionId()) || (Objects.equals(TreeEventProcessorFactory.DETAILS_VIEW_ID, representationMetadata.getDescriptionId()))) {
             /*
              * The FormDescription used for the details view can not be found by
              * IRepresentationDescriptionSearchService, but we can get away by returning a fake one with the same id as
              * no GraphQL query actually needs to see its content. We need to return *something* with the correct id
              * only to allow GraphQL resolution to continue on queries like "completionProposals" defined on
              * FormDescription.
              */
-            result = Mono.just(FAKE_DETAILS_DESCRIPTION).toFuture();
+            if (Objects.equals(PropertiesEventProcessorFactory.DETAILS_VIEW_ID, representationMetadata.getDescriptionId())) {
```

Comment:
Could you update the comment above since it does not strictly apply to the details view anymore